packages/client/components/JiraScopingSearchHistoryToggle.tsx

```diff
@@ -1,73 +1,85 @@
-  paddingRight: 8
-})
 interface Props {
-  meeting: JiraScopingSearchHistoryToggle_meeting
+  meetingRef: JiraScopingSearchHistoryToggle_meeting$key
 }
 
 const JiraScopingSearchHistoryToggle = (props: Props) => {
-  const {meeting} = props
-  const {id: meetingId, viewerMeetingMember} = meeting
-  const {togglePortal, originRef, menuPortal, menuProps} = useMenu(MenuPosition.UPPER_LEFT, {
-    loadingWidth: 200,
-    noClose: true
-  })
-  if (!viewerMeetingMember) return null
-  const {teamMember} = viewerMeetingMember
-  return (
-    <>
-      <Toggle onClick={togglePortal} ref={originRef}>
-        <SearchIcon>search</SearchIcon>
-        <DropdownIcon>expand_more</DropdownIcon>
-      </Toggle>
-      {menuPortal(
-        <JiraScopingSearchHistoryMenu
-          meetingId={meetingId}
-          menuProps={menuProps}
-          teamMember={teamMember}
-        />
-      )}
-    </>
+  const {meetingRef} = props
+  const atmosphere = useAtmosphere()
+  const meeting = useFragment(
+    graphql`
+      fragment JiraScopingSearchHistoryToggle_meeting on PokerMeeting {
+        id
+        teamId
+        viewerMeetingMember {
+          teamMember {
+            integrations {
+              atlassian {
+                jiraSearchQueries {
+                  id
+                  queryString
+                  isJQL
+                  projectKeyFilters
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    meetingRef
   )
-}
 
-export default createFragmentContainer(JiraScopingSearchHistoryToggle, {
-  meeting: graphql`
-    fragment JiraScopingSearchHistoryToggle_meeting on PokerMeeting {
-      id
-      teamId
-      viewerMeetingMember {
-        teamMember {
-          ...JiraScopingSearchHistoryMenu_teamMember
+  const {id: meetingId, teamId} = meeting
+  const {jiraSearchQueries} = meeting.viewerMeetingMember?.teamMember.integrations?.atlassian ?? {}
+
+  const searchQueries =
+    jiraSearchQueries?.map((jiraSearchQuery) => {
+      const {id, queryString, isJQL, projectKeyFilters} = jiraSearchQuery
+
+      const selectQuery = () => {
+        commitLocalUpdate(atmosphere, (store) => {
+          const searchQueryId = SearchQueryId.join('jira', meetingId)
+          const jiraSearchQuery = store.get(searchQueryId)!
+          jiraSearchQuery.setValue(isJQL, 'isJQL')
+          jiraSearchQuery.setValue(queryString, 'queryString')
+          jiraSearchQuery.setValue(projectKeyFilters as string[], 'projectKeyFilters')
+        })
+      }
+      const queryStringLabel = isJQL ? queryString : `“${queryString}”`
+      const projectFilters = projectKeyFilters
+        .map((filter) => filter.slice(filter.indexOf(':') + 1))
+        .join(', ')
+
+      return {
+        id,
+        ariaLabel: 'Remove this Jira Search Query',
+        labelFirstLine: queryStringLabel,
+        labelSecondLine: projectFilters && `in ${projectFilters}`,
+        onClick: selectQuery,
+        onDelete: () => {
```

Comment:
+1 you could make this a `const deleteQuery = ...` to be aligned with the `selectQuery`